Protocol.md

```diff
@@ -273,6 +273,8 @@ rules MAY be used for handling layout. For example, `application/x.netflix+cbor`
 * __Setup Data__: includes payload describing connection capabilities of the endpoint sending the
 Setup header.
 
+__NOTE__: A server that receives a SETUP frame that has (__R__)esume Enabled set, but does not support resuming operation, must reject the SETUP with an ERROR. 
```

Comment:
`RESUME_NOT_SUPPORTED` will give the client an opportunity to continue without resumability which I would assume will happen if the client ever wants to connect to that server.
Since, resumability is optional, I would hope a client expecting resumability can work without resumability. In such a case, leaving it on the client to continue without resume is better than forcing reconnect.